src/moderatorFrontendMain/kotlin/util/Strings.kt

```diff
@@ -240,9 +240,34 @@ object Strings {
       "Sitzplatz",
   )
 
+  val report_checkin_filter = LocalizedString(
```

Comment:
Because we might localize the word in the future to another language. or the localization might change. And then we don't want to change the type of the variable.